common/Cargo.toml

```diff
@@ -28,6 +28,7 @@ url = { version = "2" }
 sha2 = { version = "0.10.2" }
 ureq = { version = "2.2", default-features = false, features = ["json", "tls"] }
 thiserror = "1"
+walletconnect = { git = "https://github.com/xphoniex/walletconnect-rs", branch = "v0.1.0", features = ["qr"] }
```

Comment:
Let's move this below and make it optional like the other ethereum-only deps.

Reply:
https://github.com/radicle-dev/radicle-cli/commit/6bd4b556fdbdb58f9af097f15e2af011db40ed5d#diff-a51b8c3d64e9beafa9b726f1fc1c9099201507aef512f67724a1ac531573d4afR9-R57